src/cluster_legacy.c

```diff
@@ -1213,8 +1213,8 @@ void clusterInit(void) {
     if (!server.cluster_port && port > (65535 - CLUSTER_PORT_INCR)) {
         serverLog(LL_WARNING,
                   "%s port number too high. "
-                  "Cluster communication port is 10,000 port "
-                  "numbers higher than your %s port. "
+                  "Cluster communication port is 10,000 ports "
+                  "above your %s port. "
```

Comment:
This is a different heuristic. It doesn't actually suggest `above`, but I've chosen `above` in a number of cases for this PR and I may adjust my heuristic to make that suggestion as it does feel valuable for at least this repository.